packages/config/config/devices/0x010a/vitrum_i_dimmer.json

```diff
@@ -1,8 +1,8 @@
 {
 	"manufacturer": "VDA",
 	"manufacturerId": "0x010a",
-	"label": "VITRUM I",
-	"description": "Vitrum I dimmer wireless",
+	"label": "Vitrum I Dimmer",
+	"description": "Vitrum I Dimmer Wireless",
```

Comment:
If you haven't already, please take a look at our style guide. We don't repeat the label in the description. https://zwave-js.github.io/node-zwave-js/#/config-files/style-guide. Maybe a label of VITRUM I Dimmer, and a description of "Wireless 1-Channel Dimmer Switch"? Or something that makes sense.

There is also a set order for which the various elements appear. they may be messed up in the originals, but please fix them. Metadata goes at the bottom, etc.

Reply:
I'm tempted to change the manufacturer to Vitrum from VDA, as it's the brand under which the device is sold.  Which would remove Vitrum from the Label + Description as per the style guide.

The style guide says that the description should match the name under which the device is sold, so would be "I Dimmer Wireless".

So I think it's the label that is more up for debate, I can split out to SKU as the style guide suggests if you're happy with two identical files for similar products?  For example, the British I Dimmer is 01B010010 and the European I Dimmer is 01E010010.  It's always just the one letter different between the British and European.

Thoughts on all of those as proposals?

I'll sort the order of elements out.

Reply:
You can change the manufacturer to the brand it's sold under. We do that elsewhere. RE the label, just do one that makes sense. It doesn't have to be exactly the manufacturer's if they use a stupid one. We just don't want the UI to say Vitrium Vitrium I Vitrium I Dimmer Wireless, all in a row. You can make it Wireless Dimmer Switch I or whatever looks better.